clusterman/draining/queue.py

```diff
@@ -54,6 +57,15 @@
 DEFAULT_FORCE_TERMINATION = False
 DEFAULT_DRAIN_REPROCESSING_DELAY_SECONDS = 15
 DEFAULT_DRAINING_TIME_THRESHOLD_SECONDS = 1800
+EC2_FLEET_KEYS = {
+    "aws:ec2spot:fleet-request-id",
+    "aws:ec2:fleet-id",
+}
+
+
+class TerminationReason(enum.Enum):
+    SCALING_DOWN = "scaling down"
+    SPOT_INTERRUPTION = "spot interruption"
```

Comment:
suggestion: if we don't need the string values anywhere, we can always use https://docs.python.org/3/library/enum.html#using-automatic-values here instead

Reply:
I noticed that I didn't add Reason information to message in the queue.